src/OMSimulatorLib/Flags.h

```diff
@@ -129,7 +131,8 @@ namespace oms
       {"--tolerance", "", "Specifies the relative tolerance", re_double, Flags::Tolerance, false},
       {"--version", "-v", "Displays version information", re_void, Flags::Version, false},
       {"--wallTime", "", "Add wall time information for to the result file", re_bool, Flags::WallTime, false},
-      {"--workingDir", "", "Specifies the working directory", re_default, Flags::WorkingDir, false}
+      {"--workingDir", "", "Specifies the working directory", re_default, Flags::WorkingDir, false},
+      {"--doubleStep", "", "Changes the algorithm inside the WC sytems to double-step when using adaptive step-size", re_bool, Flags::DoubleStep, false}
```

Comment:
The double-step-master should be implemented as another solver method, instead of using the flag.